servers/partner-ci/utils/utils.yaml

```diff
@@ -148,7 +148,7 @@
     disabled: false
     scm:
       - git:
-          url: https://github.com/rkhozinov/jenkins-jobs.git
+          url: https://github.com/rkhozinov/jenkins-jobs
```

Comment:
separate this change from previous changes.
they don’t related, so make different prs